src/shared/components/Resources/ResourceDetails.tsx

```diff
@@ -41,20 +39,33 @@ const ResourceDetails: React.FunctionComponent<ResourceViewProps> = props => {
       }
     }
   };
+  console.log(resource && { raw: resource.raw });
```

Comment:
We really need to add that linter rule in place but in the meantime, I find it useful to go through my own file changes. These things are kind of easy to spot